README.md

```diff
@@ -36,6 +36,7 @@ Completion is available in `bash` and `zsh` shells as follows.  (Adapt these com
 # For zsh, replace bash_source with zsh_source.
 $ _AUT_COMPLETE=bash_source aut > ~/.aut-complete
 $ echo 'source ~/.aut-complete' >> ~/.bashrc
+$ exec $SHELL
```

Comment:
Note the line below states that this will affect *new shells*. We could make this more explicit. 

If we want to enable in the current shell, I'd suggest we use `source ~/.aut-complete` rather than create a child shell process.